frontend/src/libs/components/switch/switch.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  type Control,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
```

Comment:
Let's move to common types,  but add a prefix Form

Reply:
The main idea that if we decide to remove that lib, so it would be easier to handle it in one place

Reply:
ok, no problem. In input component types are imported the same way, should I change there as well?

Reply:
Yes, change all imports for these types to be from common types